src/ix_stresser/pdfbulknode.clj

```diff
@@ -0,0 +1,87 @@
+(ns ix-stresser.pdfbulknode
+  (:require [clj-http.client :as client]
+            [clojure.core.async :as async :refer [<! <!! go go-loop timeout]]
+            [clojure.data.json :as json]))
+
+(def base-files-path "./doc/")
+(def dev-url "http://localhost:3000")
+(def stag-url "http://pdfnode-staging.cloudapp.net")
+(def prod-url "http://pdfnode.cloudapp.net")
+(def existing-files ["invoice_small.json", "invoice_medium.json", "invoice_big.json"])
+
+(def invoice-on-bulk 1) ;; 1 will mean a single bulk having 3 invoices, small, medium and big
```

Comment:
I'd use a more specific name, like `document-multiplier`, something that shows this will influence the load.